trunk/NDHMS/hydro_namelists.json

```diff
@@ -73,12 +73,9 @@
         "hydro_nlist": {
             "aggfactrt": 1,
             "dxrt": 1000,
-            "io_config_outputs": 4,
-            "io_form_outputs": 2,
             "output_channelbucket_influx": 2,
             "ovrtswcrt": 0,
             "rst_typ": 0,
-            "rtout_domain": 0,
```

Comment:
Still wrapping my head around these JSON namelists for testing, but it seems like if we remove this and the default is 1 we might have a problem since long-range doesn't  do surface or subsurface routing.